RR.UnitOfWork/RR.UnitOfWork.csproj

```diff
@@ -31,7 +31,6 @@
   <ItemGroup>
     <Folder Include="Entities\ATS\" />
     <Folder Include="Interfaces\ATS\" />
-    <Folder Include="Migrations\" />
```

Comment:
Why did you delete migrations????

Reply:
I did not this happens when i run add-migration but I will try it again